src/components/widgets/ChartWidget.tsx

```
@@ -319,54 +362,48 @@ export function ChartWidget({ symbol }: { symbol?: string | undefined } = {}) {
 
-      <div className="flex gap-2 border-b border-slate-700 bg-slate-900 px-4 py-2">
+      <div className="kd-panel-header flex gap-2 px-4 py-2">
         <button
           onClick={() => toggleOverlay('vwap')}
           className={`rounded px-2 py-1 text-xs font-medium transition-colors ${
-            overlays.vwap
-              ? 'bg-[#f97316] text-white'
-              : 'bg-slate-800 text-gray-400 hover:bg-slate-700'
+            overlays.vwap ? 'kd-button-accent-active' : 'kd-button'
           }`}
         >
```

Biome (2.4.7)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)